animareflection/ui
# Input Components

## Description

##### Task link: https://trello.com/c/ktkL9sVK/164-input-components

Added `Input` and `NumberInput` components.

## Test Steps

1) Verify recipe, component, and story structure is sound.
2) Verify all tests pass (NOTE: added a `TODO` for one of the `NumberInput` tests, will have to circle back later).
3) Verify that components render appropriately in storybook and downstream apps.
